ansible/group_vars/omero-hosts.yml

```diff
@@ -6,15 +6,18 @@
 ######################################################################
 # Shared variables
 
-idr_omero_release: "0.6.1"
+idr_omero_release: "5.5.1"
 idr_omero_ice_version: "3.6"
-# Upgrades are normally disabled, set to True upgrade OMERO:
-idr_omero_upgrade: False
-# The IDR version system doesn't match the OMERO.server version
-omero_server_checkupgrade_comparator: '!='
-omero_web_checkupgrade_comparator: '!='
+idr_omero_upgrade: True
```

Comment:
The default in the omero server and web roles is to upgrade if the version doesn't match, so it should be safe to delete these variables completely:
```
$ git grep idr_omero_upgrade FETCH_HEAD
FETCH_HEAD:ansible/group_vars/omero-hosts.yml:idr_omero_upgrade: True
FETCH_HEAD:ansible/group_vars/omero-hosts.yml:omero_server_upgrade: "{{ idr_omero_upgrade }}"
FETCH_HEAD:ansible/group_vars/omero-hosts.yml:omero_web_upgrade: "{{ idr_omero_upgrade }}"
FETCH_HEAD:ansible/var-overrides/omero-ci-build.yml:idr_omero_upgrade: True
```

Reply:
See 1f3326b. I also remove the Ice variables which were redefining the role default